pkg/controller/master/setting/kubeconfigttl.go

```diff
@@ -0,0 +1,41 @@
+package setting
+
+import (
+	"fmt"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	harvSettings "github.com/harvester/harvester/pkg/settings"
+)
+
+const (
+	AuthTokenMaxTTLSettinName = "auth-token-max-ttl-minutes"
+)
+
+func (h *Handler) syncKubeconfigTTL(setting *harvesterv1.Setting) error {
```

Comment:
If I understand the code correctly, this is only applied if the setting _changes_. Are we sure it's also applied during the installation/upgrade/setup correctly?

Reply:
bootstrapSettings does the initial sync at start: https://github.com/harvester/harvester/pull/5896/files#diff-f9454369fb0cd8a32b1a568916edfe94a77af272bf63bdc5dd63c976a200cf5dR37-R40